sormas-backend/src/main/java/de/symeda/sormas/backend/util/ModelConstants.java

```diff
@@ -21,6 +21,20 @@
 
 	String PERSISTENCE_UNIT_NAME = "SormasPU";
 	String PERSISTENCE_UNIT_NAME_AUDITLOG = "auditlogPU";
-	String HINT_HIBERNATE_READ_ONLY = "org.hibernate.readOnly";
+
+
+	/**
+	 * <h2>
+	 * A query hint to make JPA entities loaded as read-only to avoid caching and dirty checks. Usage:
+	 * </h2>
+	 *
+	 *<p>
```

Comment:
Please use {@code ...} for the example in "..." and check that it gets properly formatted. Then write the Note ... in a new line (I think <br /> will be needed).